packages/core/src/lib/Table/types.tsx

```diff
@@ -104,4 +104,14 @@ export interface ITableProps {
      * Show a custom component on all pages
      */
     customDataComponent?: React.ReactNode;
+
+    /**
+     * text color of the table header; remember to use 0px gap size for proper effect
```

Comment:
`remember to use 0px gap size for proper effect`

is this part needed ? you seem to have handled it with box-shadow or am i missing something 🤔